tests/new_api/test_device.py

```diff
@@ -66,6 +63,25 @@
         validate_observables,
     )
 
+if device_name == "lightning.gpu":
```

Comment:
should we use `elif` instead of `if` and raise an error or skip for `else`?

Reply:
@maliasadi 
instead of `skip`. Would it not be better to throw an error? :thinking: